src/storage/btree_load.h

```diff
@@ -92,9 +92,9 @@
 
 /* compare two object identifiers */
 #define OIDCMP(n1, n2) \
-  ((n1).volid == (n2).volid \
-   && (n1).pageid == (n2).pageid \
-   && (n1).slotid == (n2).slotid)
+  ((n1).pageid == (n2).pageid \
```

Comment:
Why did you do this reordering?

Reply:
This macro only checks if the OID is the same.
A total of 3 comparisons (volid, pageid, slotid) are required, and it is efficient to first compare those that are unlikely to match.
For this reason, we checked the pageid, which has the widest range of values, first.